hack/generators/generate.py

```diff
@@ -0,0 +1,320 @@
+#!/usr/bin/env python
+from __future__ import print_function
+import re
+import jinja2
+import optparse
+import logging
+import json
+
+
+def setOptions():
+
+    parser = optparse.OptionParser("%prog ")
```

Comment:
Yeah, serving in the next PR I hope unless some crazy idea comes first and I will "just" push another commit.